src/components/Loaders/cardShimmer/index.js

```diff
@@ -2,14 +2,20 @@ import React from 'react';
 import classNames from './cardShimmer.module.scss';
 
 const CardShimmer = () => (
-
   <div className={`${classNames.card} ${classNames.br}`}>
     <div className={classNames.wrapper}>
-      <div className={`${classNames.title} ${classNames.br} ${classNames.animate}`} />
-      <div className={`${classNames.create} ${classNames.br} ${classNames.animate}`} />
-      <div className={`${classNames.comment} ${classNames.br} ${classNames.animate}`} />
-      <div className={`${classNames.comment} ${classNames.br} ${classNames.animate}`} />
-      <div className={`${classNames.button} ${classNames.animate}`} />
+      <div
+        className={`${classNames.title} ${classNames.br} ${classNames.animate}`}
+      />
+      <div
+        className={`${classNames.create} ${classNames.br} ${classNames.animate}`}
+      />
+      <div
+        className={`${classNames.comment} ${classNames.br} ${classNames.animate}`}
+      />
+      <div
+        className={`${classNames.comment} ${classNames.br} ${classNames.animate}`}
+      />
```

Comment:
This is just an indentation change, let's revert it back

Reply:
I'll fix the issues, but I think the shimmer is already responsive, see this: 

https://user-images.githubusercontent.com/61474917/123444884-0145fd80-d5f5-11eb-924c-4a14f9c1a8e6.mp4

This is how it looks in full screen mode

Reply:
Also, do you want me to nest all the css in the in `cardShimmer.module.scss` into `sass` style ?

Reply:
- Sure please nest the `scss`
- Can you make the shimmer occupy full width in mobile?

![RDSStatusShimmer](https://user-images.githubusercontent.com/26569942/125733301-ca02c518-a9f8-462d-ac31-a32dcd93471d.gif)

Reply:
This is how it looks now: 

https://user-images.githubusercontent.com/61474917/127209605-176c5cd7-d31a-4d5b-a8e8-03dc8096d880.mp4